tests/dashboardPage.ts

```diff
@@ -0,0 +1,72 @@
+import { Page, Locator } from "@playwright/test";
+
+export default class DashboardPage {
+  pageDomain: Page;
+
+  firstNameText: string = "Malek";
+  lastNameText: string = "Tubaishat";
+  middleNameText: string = "abdel";
+  employeeID: string;
+
+  bannerTitle: Locator;
+  firstNameTextBox: Locator;
+  middleNameTextBox: Locator;
+  lastNameTextBox: Locator;
+  submitButton: Locator;
+  employeeTitle: Locator;
+  employeeIDTextBox: Locator;
+  employeeIDTextBoxInEmployeeList: Locator;
+  employeeListButton: Locator;
+  employeeSearchButton: Locator;
+  employeeDeleteButton: Locator;
+  employeeDeleteConfirmation: Locator;
+
+  constructor(pageDomain: Page) {
+    if (pageDomain != null) {
+      this.pageDomain = pageDomain;
+      this.employeeID = this.getRandomSixDigitNumber().toString();
+      this.initializeLocators();
+    }
+  }
+
+  async initializeLocators() {
+    try {
+      this.bannerTitle = this.pageDomain.locator(
```

Comment:
please fix the indentation in the file and add the locator's name like 
this.bannerTitle= then add a space after then add the value of the locator